_includes/specials/taproot/en/10-ln-with-taproot.md

```diff
@@ -0,0 +1,194 @@
+  multisignatures]).
+
+In principle, a pre-taproot channel can be upgraded to support PTLCs without
+closing and reopening the channel.  Existing channels can host PTLCs by
+creating an offchain transaction that spends the existing non-taproot
+funding output to a taproot output containing a PTLC.  Thus getting
+support for PTLCs over LN does not require any cost to users
+beyond each node and its channel peers upgrading their software.
+
+However, to actually use PTLCs, *every* forwarding node from the spender
+to the receiver must support PTLCs.  This means PTLC support may remain
+largely unused until a sufficient number of nodes have upgraded.  They
+don't all necessary need to use the same protocol (there could be
+multiple PTLC protocols), but they all must support some PTLC protocol.
+Having to support multiple PTLC protocols would be an added maintenance
+burden and I *hope* we do not have too many such protocols (ideally just
+one).
+
+### P2TR Channels
+
+One solution for improving the decorrelation between the base layer and
+the LN layer has been [unpublished channels][topic unannounced channels]---channels whose
+existence isn't gossiped on LN.
+
+Unfortunately, every LN channel is a 2-of-2, and in the current
+pre-taproot Bitcoin, every 2-of-2 is *openly* coded.  LN is the
+most popular user of 2-of-2 multisignature, so any blockchain explorer
+can see a 2-of-2 being spent and guess with fairly good probability that
+this is a LN channel being closed.  The funds can then be traced
+from there, and if it goes to another P2WSH output, then that is likely
+to be *another* unpublished channel.  Thus, even unpublished channels
+are identifiable onchain once they are closed, with some level of false
+positives.
+
+Taproot, by using [schnorr signatures][topic schnorr signatures], allows for n-of-n to look exactly
+the same as 1-of-1.  With some work, even [k-of-n][topic threshold signature] will also look the same
+as 1-of-1 (and n-of-n).  We can then propose a feature where an LN
+channel is backed by a P2TR UTXO, i.e. a
+P2TR channel, which increases the *onchain* privacy of
+unpublished channels.[^two-to-tango]
+
+<!-- P2WSH 2-of-2: OP_0 <sig> <sig> <2 <key> <key> 2 OP_CMS>
+             219 =   1 + 1+72 +1+72 +1+1+1+33+1+33+1+1
+             54.75 = 219/4
+     P2TR: <sig>
+             64
+             16 = 64/4
+
+    Comparsion:
+      38.75 = 54.75 - 16
+      ~70% = 1 - 16/54.75
+-->
+
+This (rather small) privacy boost also helps published channels as well.
+Published channels are only gossiped until they are closed, so somebody
+trying to look for published channels will not be able to learn about
+*historical* channels.  If a surveillor wants to see every published
+channel, it has to store all that data itself and cannot rely on any
+kind of "archival" node.
+
+In addition, taproot keypath spends are 38.5 vbytes (70%) smaller than
+LN's existing P2WSH spends.  Unfortunately, you **cannot upgrade
```

Comment:
```suggestion
In addition, taproot keypath spends are 39.25 vbytes (40%) smaller than
LN's existing P2WSH spends.  Unfortunately, you **cannot upgrade
```

This only compares the witness stack, not the full input size. Counting the `prevout`, `scriptSig`, and `nSequence`, a P2TR keypath input is **57.5 vB**, and a 2-of-2 P2WSH input should be **96 vB** (95.5 vB assuming signature grinding). This comes out to a 40% cost reduction for inputs.